@xen-orchestra/web/src/components/hosts/HostsTable.vue

```diff
@@ -0,0 +1,251 @@
+<template>
+  <div class="hosts-table">
+    <UiTitle>{{ t('hosts') }}</UiTitle>
+    <div class="container">
+      <div class="table-actions">
+        <UiQuerySearchBar @search="value => (searchQuery = value)" />
+        <UiTopBottomTable :selected-items="0" :total-items="0">
+          <UiTablePagination v-bind="paginationBindings" />
+        </UiTopBottomTable>
+      </div>
+      <VtsDataTable :no-data-message="hosts.length === 0 ? t('no-hosts-detected') : undefined" :has-error :is-ready>
+        <template #thead>
+          <tr>
+            <template v-for="column of visibleColumns" :key="column.id">
+              <th v-if="column.id === 'checkbox'" class="checkbox">
+                <div v-tooltip="t('coming-soon')">
+                  <UiCheckbox disabled :v-model="areAllSelected" accent="brand" @update:model-value="toggleSelect" />
+                </div>
+              </th>
+              <th v-else-if="column.id === 'more'" class="more">
+                <UiButtonIcon v-tooltip="t('coming-soon')" icon="fa:ellipsis" accent="brand" disabled size="small" />
+              </th>
+              <th v-else>
```

Comment:
```suggestion
              <th>
```

due to the recent changes, I think we can remove columns `checkbox ` and `more`